packages/attention/index.js

```diff
@@ -342,27 +346,27 @@ class WarpAttention extends kebabCaseAttributes(WarpElement) {
   }
 
   render() {
-    return html`
-      <div class=${ifDefined(this.className ? this.className : undefined)}>
-        ${this.placement === 'right-start' || this.placement === 'right' || this.placement === 'right-end' || this.placement === 'bottom-start' || this.placement === 'bottom' || this.placement === 'bottom-end' // Attention's and its arrow's visual position should be reflected in the DOM
-          ? html`
-              <slot name="target"></slot>
-
-              <div id="attention" role="${this.tooltip ? 'tooltip' : 'img'}" aria-label="${this.defaultAriaLabel()}" class="${this._wrapperClasses}">
-                ${this._arrowHtml}
-                <slot name="message"></slot>
-                ${this.canClose ? this._closeBtnHtml : nothing}
-              </div>
-            `
-          : html`
-              <div id="attention" class="${this._wrapperClasses}">
-                <slot name="message"></slot>
-                ${this._arrowHtml} ${this.canClose ? this._closeBtnHtml : nothing}
-              </div>
-              <slot name="target"></slot>
-            `}
-      </div>
-    `;
+    return html` ${this.callout || (this._targetEl !== undefined && !this.callout)
```

Comment:
I suppose this has something to do with changes introduced in the core library (https://github.com/warp-ds/core/pull/16), but I'm not sure what exactly. If this check is necessary, maybe we could turn it around for better readability, like it was done in the case of Toast (https://github.com/warp-ds/elements/blob/next/packages/toast/toast.js#L152)?

Reply:
Updated in this commit: [c10e8ce](https://github.com/warp-ds/elements/pull/149/commits/c10e8ce967df3e229b1486aadd9b546a33242fb1) (I fixed the lint issue in this commit 🙈)